app/src/main/java/com/babylon/wallet/android/presentation/account/settings/AccountSettingsNav.kt

```diff
@@ -13,24 +13,38 @@ import androidx.navigation.navArgument
 import com.google.accompanist.navigation.animation.composable
 
 @VisibleForTesting
-internal const val ARG_ADDRESS = "arg_address"
+internal const val ARG_ACCOUNT_SETTINGS_ADDRESS = "arg_account_settings_address"
+internal const val ARG_ACCOUNT_SETTINGS_NAME = "arg_account_settings_name"
 
-internal class AccountSettingsArgs(val address: String) {
-    constructor(savedStateHandle: SavedStateHandle) : this(checkNotNull(savedStateHandle[ARG_ADDRESS]) as String)
+internal class AccountSettingsArgs(
+    val address: String,
+    val name: String
+) {
+    constructor(savedStateHandle: SavedStateHandle) : this(
+        checkNotNull(savedStateHandle[ARG_ACCOUNT_SETTINGS_ADDRESS]) as String,
+        checkNotNull(savedStateHandle[ARG_ACCOUNT_SETTINGS_NAME]) as String
+    )
 }
 
-fun NavController.accountSettings(address: String) {
-    navigate("account_settings_route/$address") {
+fun NavController.accountSettings(
+    address: String,
+    name: String
+) {
+    navigate("account_settings_route/$address/$name") {
         launchSingleTop
```

Comment:
this is property, so shouldn't it be either = true or false

Reply:
> The account address would suffice and the view model can fetch the whole account's details.

sometimes I am wondering why I am doing such silly implementation although I know that this is what I would have said if I would review something similar. 🤦🏽

Reply:
>this is property, so shouldn't it be either = true or false

hmm what do you mean here?

Reply:
oh I see. Thanks I set it to true.